lib/app.js

```diff
@@ -8,10 +9,20 @@ const app = express();
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(
+    cors({
+      origin: [
+        'https://alchemy-shopping-front-end-demo.netlify.app',
+        'http://localhost:7890',
+      ],
+      credentials: true,
+    })
+  );
 
 // App routes
 
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
FYI, since all the routes in items use the authenticate middleware, you can add it here directly
`app.use('/api/v1/items', authenticate, require('./controllers/items'));`